frontend/talentsearch/src/js/api/userOperations.graphql

```diff
@@ -4,3 +4,28 @@ query getMe {
     lastName
   }
 }
+
+query getAboutMe {
```

Comment:
So this is definitely the way it has been done through most of the codebase so I'm not surprised you've done it this way.  We just started moving the graphql operations that are specific to one component into a file in the directory that holds all the other component files like this: https://github.com/GCTC-NTGC/gc-digital-talent/pull/2323/files#diff-8df74b18c99910aa5360f95ff5370238e3bd617cdbb2b849bd54063cd8d2119d  Could you give this a try?